src/ca/noxid/lab/EditorApp.java

```diff
@@ -2061,6 +2061,11 @@ public void actionPerformed(ActionEvent e) {
 			if (exeData == null) {
 				return;
 			}
+			String message = Messages.getString("MapDeleteConfirm.0") + selectedMapStr + Messages.getString("MapDeleteConfirm.1");
+			int choice = JOptionPane.showConfirmDialog(this, message, Messages.getString("MapDeleteConfirm.2"), JOptionPane.YES_NO_OPTION);
+			if (choice == 1) {
```

Comment:
This should use the appropriate named constant in `JOptionPane`, and instead of checking for a "no" you should instead check that the response "isn't yes", because the user could close (or cancel?) the dialog and those should all have the same behaviour as "no" in this case.